server/middlewares/validations/ReportValidation.js

```diff
@@ -0,0 +1,80 @@
+import Validator from 'validatorjs';
+
+/**
+ * @class ReportValidation
+ */
+class ReportValidation {
+  /**
+   * @description validate report input
+   *
+   * @param {object} req
+   * @param {object} res
+   * @param {func} next
+   *
+   * @return {void}
+   */
+  static validateReport(req, res, next) {
+    const { details, title } = req.body;
+
+    const data = {
+      title,
+      details,
+    };
+
+    const rules = {
+      title: 'required|max:80|min:3',
+      details: 'required|max:1000|min:3',
+    };
+
+    const message = {
+      'required.body': ':attribute field cannot be empty',
+      'max.body': 'The :attribute is too long. Max length is :max characters.',
+    };
+
+    const validation = new Validator(data, rules, message);
+
+    if (validation.passes()) {
+      return next();
+    }
+
+    return res.status(400).json({
+      errors: validation.errors.all(),
+    });
+  }
+
+  /**
+   * @description validate id for report
+   *
+   * @param {object} req
+   * @param {object} res
+   * @param {func} next
+   *
+   * @return {void}
+   */
+  static validateId(req, res, next) {
+    const { reportId } = req.params;
+
+    Validator.register(
+      'positiveInt', value => value > 0,
+      'The report :attribute must be a positive integer',
+    );
+
+    const validation = new Validator({
+      reportId,
+    }, {
+      reportId: 'required|integer|positiveInt',
+    }, {
+      'integer.id': 'The report :attribute must be an integer',
```

Comment:
I believe what you want to write a custom message for here is the 'reportId' and not 'id'.